deploy.php

```diff
@@ -124,12 +124,15 @@ function () {
 
     $executedMigrations = explode("\n", run('cat executed_migrations'));
 
-    // var_dump((int) run('find {{release_path}}/migrations/* -maxdepth 0 -type d | wc -l'));
+    cd('{{release_path}}/migrations/');
 
-    // die;
+    $dirs = explode(',', run('find {{release_path}}/migrations/* -maxdepth 0 -type d | tr "\n" ","'));
+    var_dump($dirs);
```

Comment:
Waarom commit je uw debug code? Je kan dat toch allemaal perfect testen zonder te commiten?

Reply:
true, er zal in mijn hoofd wel een reden geweest zijn maar ik herinner het me niet meer